statistical_validation_run/artifacts.py

```
@@ -0,0 +1,635 @@
+    lines += [
+        "",
+        f"> {disclaimer}",
+        "",
+        "## Inclusive outcomes (every close method, every loss)",
+        "",
+        "- **n_closes**: "
+        f"{ta.get('n_closes')} (distinct close rows, all methods: "
+        f"{', '.join(methods) or 'none'})",
+        "- **total_realized_pnl**: "
+        f"${portfolio.get('realized_pnl'):.2f} "
+        f"({'losses folded in' if (portfolio.get('realized_pnl') or 0) < 0 else 'net of all losses'})",
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)